verification/curator-service/ui/src/components/fixtures/fullCase.json

```diff
@@ -110,16 +110,16 @@
         "travel": [
             {
                 "dateRange": {
-                    "start": "2020-01-10",
+                    "start": "2020-02-10",
```

Comment:
Why this change? It makes it an invalid date range

Reply:
because 2020-01-10 was already used somewhere else and findByText was complaining that it now found two occurences, we don't really check that start < end but yes it makes sense to have it valid, fixed.